inline_scan.sh

```diff
@@ -581,6 +584,9 @@ cleanup() {
         if [[ -d "/tmp/sysdig" ]]; then
             rm -rf "/tmp/sysdig"
         fi
+        if [[ "${v_flag-""}" ]] && [[ -d "${VOLUME_PATH}" ]]; then
+            rm -rf "${VOLUME_PATH}"
```

Comment:
This is scary. The help text doesn't mention anything about the directory being deleted, and `rm -rf $VARIABLE` can have "interesting" side effects, I'd suggest to remove this.

Reply:
You're totally right!
I've removed those lines